File: Algorithms/newYearChaos.js
// Complete the minimumBribes function below.
function minimumBribes(q) {
    let count = 0

    for(let i = q.length; i-- >0;){
        if(q[i] - (i+1) > 2){
            count = 'Too chaotic'
            break;
        }
        for(let j = Math.max(q[i]) - 2; j < i; j++){
            if(q[j] > q[i]){
                count+=1
            }
        }
    }
    console.log(count)

}

File: Interview Preparation/Array/Python/newYearChaos.py
# Complete the minimumBribes function below.
def minimumBribes(q):
    count = 0
    for i in range(2):
        for j in range(len(q) - 1, 0, -1):
            if q[j] < q[j-1]:
                q[j], q[j-1] = q[j-1], q[j]
                count += 1

    if q == sorted(q):
        print(count)
    else:
        print('Too chaotic')